overlays/dnf-master/overlay.yml

```diff
@@ -9,6 +9,7 @@ components:
   - name: libsolv
     git:
       src: github:openSUSE/libsolv.git
+      freeze: 335f469bf6e3109d4973e55a3a86a274adb0c21d
```

Comment:
this freeze refers to distgit commit, not upstream

Reply:
whoops, wrong section. Can you fix it?

Reply:
Yes, I'll do it